api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -415,15 +441,29 @@ message HeaderMutation {
   repeated string remove_headers = 2;
 }
 
-// The body response message corresponding to FULL_DUPLEX_STREAMED body mode.
+// The body response message corresponding to ``FULL_DUPLEX_STREAMED`` or ``GRPC`` body modes.
```

Comment:
Each field in this message now has a comment above it to indicate that it means semantically different things based on a field contained in a different message.

This is saying to me that it belongs as a different message in a oneof at a higher level.